### PR TITLE
`deep-reblame` - Restore feature 

### PR DESCRIPTION
**Pull request by kovsu**
close: #6838

## Test URLs

https://github.com/yakov116/TestR/blame/master/asf

## Screenshot

截屏2023-11-27 14 33 46

### CODE REVIEW

**Comment by fregante** [comment]
Just the second place

File: source/features/deep-reblame.tsx [hunk 1]
```diff
@@ -60,22 +58,12 @@ async function redirectToBlameCommit(event: DelegateEvent<MouseEvent, HTMLAnchor
...
-	} else {
-		$('.blob-reblame', hunk)!.append(
-			<button
-				type="button"
-				aria-label="View blame prior to this change (extracts commits from this PR first)"
-				className="reblame-link btn-link no-underline tooltipped tooltipped-e d-inline-block pr-1 rgh-deep-reblame"
-			>
-				<VersionsIcon/>
-			</button>,
-		);
```
Comment (fregante) [inline source/features/deep-reblame.tsx:78]: ?

This piece of code added the button to hunks that don't already have one.

For example the first PR commit in a file.

Test URL:

https://github.com/refined-github/refined-github/blame/main/source/github-helpers/heat-map.css

Every line here should have that button.

**Comment by kovsu** [comment]
@fregante Hi, I wonder what the selector means here. What elements does it select before? Because it was broken now.
`blameUrl.hash = 'L' + $('.js-line-number', blameHunk)!.textContent;`

**Comment by fregante** [comment]
What do you mean? The selector mentions "line number", it selects the line number.

**Comment by kovsu** [comment]
🥲 I can't fix it.

**Pull request closed without merging**